.bumpversion.cfg

```diff
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.5.3
+current_version = 1.0.0-rc1
```

Comment:
I think this will be normalized to 1.0.0rc1. Maybe change the version identifier to match to avoid confusion